Documentation/developer-guide.md

```diff
@@ -23,6 +23,28 @@ We use the following workflow for building and testing features and fixes.
 
 You first need to [Fork](https://github.com/dotnet/corefx/wiki/Checking-out-the-code-repository#fork-the-repository) and [Clone](https://github.com/dotnet/corefx/wiki/Checking-out-the-code-repository#clone-the-repository) this WPF repository. This is a one-time task.
 
+
+### Running DRTs locally ###
+In order to run the set of DRTs on your local machine, pass the `-test` parameter to the `build.cmd` script. At the end of the run, you should see something like this:
+
+```
+  A total of 1 test Infos were processed, with the following results.
+   Passed: 1
+   Failed (need to analyze): 0
+   Failed (with BugIDs): 0
+   Ignore: 0
+
+```
+If there were any failures, you can cd into $(RepoRoot)\artifacts\test\$(Configuration)\$(Platform)\Test and run the tests manually with the `/debugtests` flag. This will open a cmd window, where you can open the test executable in Visual Studio and debug it. For example, to debug DrtXaml.exe you would enter this into the command window:
+
+`"%ProgramFiles%\Microsoft Visual Studio\2019\Preview\Common7\IDE\devenv.exe" DrtXaml.exe`
```

Comment:
There are actually cmd scripts that come with QV for this.  When the debug prompt is up, you just type "vs" or "windbg" and it will load the appropriate debugger.  The instructions for this are in the debug window itself, but might make sense to mention it here.

Reply:
those don't actually seem to work. that only launches vs in a way that debugs sti

Reply:
STI runs the test, that's the point.

Reply:
I see, that's probably the issue.  You can use windbg with attach to child-process to pick it up, but makes sense.